numba/tests/test_inplace.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import print_function
+import numba.unittest_support as unittest
+from numba import types, utils, jit
+from numba.tests import usecases
+import numpy as np
+
+@jit(nopython=True)
+def add_5(x):
+    x += 5
+
+
+class TestInplaceAdd(unittest.TestCase):
+
+    def test_int(self):
+        arr = np.arange(10)
+        add_5(arr)
+        res = np.arange(10) + 5
+        for (i, j) in zip(arr, res):
```

Comment:
Note you don't have to write the loop by hand. You can use the `assertPreciseEqual()` helper in `test.support.TestCase` or, alternatively, one of Numpy's testing methods: http://docs.scipy.org/doc/numpy/reference/routines.testing.html